lib/cli.js

```diff
@@ -44,7 +44,8 @@ const SUPPORTED_SERVICES = [
     ' phantomjs - https://github.com/cognitom/wdio-phantomjs-service',
     ' static-server - https://github.com/LeadPages/wdio-static-server-service',
     ' visual-regression - https://github.com/zinserjan/wdio-visual-regression-service',
-    ' webpack - https://github.com/leadpages/wdio-webpack-service'
+    ' webpack - https://github.com/leadpages/wdio-webpack-service',
+    ' webpack-dev-server - https://gitlab.com/Vinnl/wdio-webpack-dev-server-service'
```

Comment:
I like seeing all these packages getting build but in this case wouldn't it make sense to add the functionality of `webpack-dev-server` to the webpack service?